.gitignore

```diff
@@ -1,8 +1,5 @@
-vinca.yaml
 recipes/
-*.bat
-*.sh
-*.ps1
```

Comment:
Do we really want these?

Reply:
From what I understand, that was done as vinca was creating temporary files in the root folder. After https://github.com/RoboStack/vinca/pull/84 this is not the case anymore, so I thought to clean that up? However, we do not neeed to actually commit any of those files, so I can also revert.